README.md

```diff
@@ -1,82 +1,103 @@
 [![Coverage Status](https://coveralls.io/repos/github/spyder-ide/spyder-vim/badge.svg?branch=master)](https://coveralls.io/github/spyder-ide/spyder-vim?branch=master)
 
 ----
 
-## Important Announcement: Spyder is unfunded!
-
-Since mid November/2017, [Anaconda, Inc](https://www.anaconda.com/) has
-stopped funding Spyder development, after doing it for the past 18
-months. Because of that, development will focus from now on maintaining
-Spyder 3 at a much slower pace than before.
-
-If you want to contribute to maintain Spyder, please consider donating at
-
-https://opencollective.com/spyder
+# Overview
 
-We appreciate all the help you can provide us and can't thank you enough for
-supporting the work of Spyder devs and Spyder development.
+Spyder Plugin for executing Vim commands inside the code editor. Although it does not currently support many VIM commands, it will be developed gradually.
 
-If you want to know more about this, please read this
-[page](https://github.com/spyder-ide/spyder/wiki/Anaconda-stopped-funding-Spyder).
+<img src="/doc/example.gif" width="600">
 
-----
+| Category     | supported commands                                                            |
+| ------------ | ----------------------------------------------------------------------------- |
+| Movement     | h, j, k, l, w, b, e, space, backspace, return, $, 0, ^, G, gg, zz, H, L, M, % |
+| Change       | x, r, o, O, u, d, dd, dw, D, c, cc, cw, J, ~, <, >, <<, >>                    |
+| Copy & Paste | yy, yw, y$, p, P                                                              |
+| Search       | /, ?, n, N, f, F                                                              |
+| mode         | i, I, a, A, v, V                                                              |
+| Register     | -, 0, 1, unamed                                                               |
+| File         | ZZ, gt, gT, :w, :q, :wq, :n, :e                                               |
 
 ## Installation
+
 To install this plugin, you can use either ``pip`` or ``conda`` package managers, as it follows:
 
 Using pip:
+
 ```
 pip install spyder-vim
 ```
 
 Using conda:
+
 ```
 conda install spyder-vim -c spyder-ide
 ```
 
 ## Dependencies
+
 This project depends on [Spyder](https://github.com/spyder-ide/spyder).
 
 ## Changelog
+
 Visit our [CHANGELOG](CHANGELOG.md) file to know more about our new features and improvements.
 
 ## Development and contribution
+
 Do you want to request a new keybind for the plugin? Please submit it to our [Commands](https://github.com/spyder-ide/spyder-vim/issues/1) issue page. Feel free to open a PR to implement it.
 
 To start contributing to this project, you can execute ``pip install -U .`` to test your changes on Spyder. We follow PEP8 and PEP257 style guidelines.
 
-# Overview
-![alt tag](/doc/example.gif)
-
-
 ## Contributing
 
 Everyone is welcome to contribute!
```

Comment:
Checking again, maybe we can merge the `Development and contribution` and `Contributing` sections into a sigle one and also add the link for the contributing guide at https://github.com/spyder-ide/spyder-vim/blob/master/CONTRIBUTING.md

The text could be something like:

```
Do you want to request a new keybind for the plugin? Please submit it to our [Commands](https://github.com/spyder-ide/spyder-vim/issues/1) issue page. Feel free to open a PR to implement it.

To start contributing to this project, please check our [contributing guide](https://github.com/spyder-ide/spyder-vim/blob/master/CONTRIBUTING.md) to setup a development environment and be able to test your changes on Spyder. We follow PEP8 and PEP257 style guidelines.

Everyone is welcome to contribute!

```